### PR TITLE
Add compile android arm native lib

### PR DESCRIPTION
**Pull request by psini**
This pull add the compile of sqlitejdbc for android arm. 
Thank you

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -98,6 +98,9 @@ $(NATIVE_DLL): $(SQLITE_OUT)/$(LIBNAME)
 	@mkdir -p $(NATIVE_TARGET_DIR)
 	cp $< $(NATIVE_TARGET_DIR)/$(LIBNAME)
 
+android-arm: $(SQLITE_UNPACKED) jni-header
+	./docker/dockcross-android bash -c 'make clean-native native CROSS_PREFIX=/usr/arm-linux-androideabi/bin/arm-linux-androideabi- OS_NAME=Android OS_ARCH=arm'
```
Comment (xerial) [inline Makefile]: A concern here is OS_NAME (Android) should match JVM system property `os.name` of DalvikVM for Android.

**Comment by xerial** [comment]
To check the system is android or not, we need to run `System.getProperty("java.runtime.name", "").toLowerCase().contains("android")` within OSInfo.java

**Comment by xerial** [comment]
@psini If you can fix OSInfo.java to add support for android OS, it would be great. But if you can wait for a week, I can do it on behalf of you.

**Comment by psini** [comment]
I will try to make myself

Il 31 Dic 2016 06:59, "Taro L. Saito" <notifications@github.com> ha scritto:

**Comment by psini** [comment]
Done make me know if i have to do other change

File: src/main/java/org/sqlite/util/OSInfo.java [hunk 2]
```diff
@@ -162,6 +162,9 @@ else if (osName.contains("Linux")) {
         else if (osName.contains("AIX")) {
             return "AIX";
         }
+        else if (osName.contains("Android")) {
```
Comment (xerial) [inline src/main/java/org/sqlite/util/OSInfo.java]: @psini Did you check the actual value of `os.name` JVM property under Android?
I think it's just Linux, so this code wouldn't work as expected

**Comment by xerial** [comment]
I made changes in #193. Let me close this PR. Thanks.

**Pull request closed without merging**